CONTRIBUTING.md

```diff
@@ -20,6 +21,14 @@ Want to contribute to Open Source Friday? That's great! Here are a couple of gui
 
 You can find in-depth instructions to install the correct Ruby version, Postgres, and to set up the database in our [README](https://github.com/ossfriday/ossfriday/blob/master/README.md#getting-started).
 
+## Localisation
+
+ * Fork the project.
+ * Copy config/locale/en.yml to target locale file ex. config/locale/es.yml.
+ * Add target language in config/initializers/locale.rb:I18n.available_locales = %i[en, ja]
+ * Translate messages in locale file.
+ * Send a pull request.
```

Comment:
May want to update based on the new README changes i.e. note creating a branch etc.